.neutrinorc.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
           {
             href: './static/favicon.png',
             rel: 'shortcut icon',
+          },
+          {
+            href: 'https://cdn.materialdesignicons.com/2.7.94/css/materialdesignicons.min.css',
```

Comment:
I feel like there is no need to add materialdesignicons CDN, afaik, we had [mdi-react](https://github.com/levrik/mdi-react) library from which you can add all the icons using something like `<TestTubeIcon />`

Reply:
Agreed. We can specify the icons in the yaml file and dynamically import the icons. I will try to send a patch for that :)